.github/workflows/test.yml

```
@@ -17,30 +17,38 @@ jobs:
     - name: Start container
       run: |
         docker run -it --name CI_container -v ${GITHUB_WORKSPACE}:/Package -v /cvmfs:/cvmfs:shared -d ghcr.io/aidasoft/centos7:latest /bin/bash
-#    - name: CMake Configure
-#      run: |
-#        docker exec CI_container /bin/bash -c 'cd Package;\
-#         mkdir -p build install;\
-#         git submodule init;\
-#         git submodule update;\
-#         source ${{ matrix.SETUP }};\
-#         cd build;\
-#         cmake -DCMAKE_INSTALL_PREFIX=../install -DCMAKE_CXX_STANDARD=17  -DCMAKE_CXX_FLAGS=" -fdiagnostics-color=always " -G Ninja ..;'
-#    - name: Compile 
-#      run: |
-#        docker exec CI_container /bin/bash -c 'cd ./Package;\
-#        source ${{ matrix.SETUP }};\
-#        cd build;\
-#        ninja -k0;'
-#    - name: Install
-#      run: |
-#        docker exec CI_container /bin/bash -c 'cd ./Package;\
-#          source ${{ matrix.SETUP }};\
-#          cd build;\
-#          ninja -k0 install;'
-#    - name: Test
-#      run: |
-#        docker exec CI_container /bin/bash -c 'cd ./Package;\
-#        source ${{ matrix.SETUP }};\
-#        cd build;\
-#        ninja -k0 && ctest --output-on-failure;'
+    - name: CMake Configure
+      run: |
+        docker exec CI_container /bin/bash -c 'cd Package;\
+         mkdir -p build install;\
+         git submodule init;\
+         git submodule update;\
+         source ${{ matrix.SETUP }};\
+         cmake -S . -B build -GNinja -DCMAKE_INSTALL_PREFIX=$PWD/install -DCMAKE_CXX_STANDARD=17;'
+
+    - name: Build Open Data Detector
+      run: |
+        docker exec CI_container /bin/bash -c 'cd ./Package;\
+        source ${{ matrix.SETUP }};\
+        git clone https://gitlab.cern.ch/acts/OpenDataDetector.git;\
+        mkdir build_odd;\
+        cmake -S OpenDataDetector -B build_odd -GNinja -DCMAKE_CXX_STANDARD=17 -DCMAKE_INSTALL_PREFIX=$PWD/install;\
+        cmake --build build_odd;\
+        cmake --install build_odd;'
+
+    - name: Build package
+      run: |
+        docker exec CI_container /bin/bash -c 'cd ./Package;\
+        source ${{ matrix.SETUP }};\
+        source $PWD/install/bin/this_odd.sh;\
+        cmake --build build;\
+        cmake --install build;'
+
+    - name: Test
+      run: |
+        docker exec CI_container /bin/bash -c 'cd ./Package;\
+        source ${{ matrix.SETUP }};\
+        export ODD_XML=$PWD/install/share/OpenDataDetector/xml/OpenDataDetector.xml;\
```

I'd propose to agree on one environment variable to find the opendatadetector xmls - usually we use PACKAGENAME in all caps, that way one doesn't need to remember any peculiarities in spelling, which really becomes important when the number of packages increases
```suggestion
        export OPENDATADETECTOR=$PWD/install/share/OpenDataDetector/;\
```